apps/server/src/config/swagger.ts

```diff
@@ -14,7 +14,7 @@ const swaggerDefinition = {
 const options = {
 	swaggerDefinition,
 	// Paths to files containing OpenAPI definitions
-	apis: ['./src/routes/*.ts'],
+	apis: ['./src/routes/*.ts', './src/routes/swagger/*.yml'],
```

Comment:
this generates the swagger output file based on swagger annotations and YML files.